src/components/common/ChartLegend.tsx

```diff
@@ -0,0 +1,14 @@
+import { LabelProps } from "../../@types/components/commonTypes";
+
+const ChartLegend = (props: LabelProps) => {
+  return (
+    <div className={`flex items-center ${props.containerClassName}`}>
+      <span
+        className={`h-[15px] w-[15px] rounded-[50%] inline-block ${props.circleColor}`}
```

Comment:
1. can't we add 15px to tailwind.config?
2. use rounded-full if it is for circle
3. ${props.circleColor || ""}

Reply:
Done.